xml/System/IAsyncDisposable.xml

```diff
@@ -15,8 +15,34 @@
   </AssemblyInfo>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides a mechanism for releasing unmanaged resources asynchronously.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[  
+
+##Remarks
+
+In .NET, classes that own unmanaged resources usually implement <xref:System.IDisposable> interface to provide a mechanism for releasing unmanaged resources synchronously. However, in some cases they need to provide an asynchronous mechanism for releasing unmanaged resources in addition to (or instead of) the synchronous one. Providing such mechanism enables the consumer to perform resource-intensive dispose operations without blocking the main thread of a GUI application for a long time.
```

Comment:
```suggestion
In .NET, classes that own unmanaged resources usually implement the <xref:System.IDisposable> interface to provide a mechanism for releasing unmanaged resources synchronously. However, in some cases they need to provide an asynchronous mechanism for releasing unmanaged resources in addition to (or instead of) the synchronous one. Providing such a mechanism enables the consumer to perform resource-intensive dispose operations without blocking the main thread of a GUI application for a long time.
```